CONTRIBUTING.md

```diff
@@ -129,6 +130,34 @@ We generally review for:
 To speed the process along, we may merge small changes such as spelling and formatting
 into your branch.
 Otherwise, we make suggestions and work with you to finalize the changes.
+<!-- vale Platform.first-person = YES -->
+
+### Review comment style
+
+To make comments clearer, each comment should have an emoji label describing its purpose.
+The following labels are used:
+
+| Emoji | `:code:`              | Meaning      | Description |
+| ----- | --------------------- | ------------ | ----------- |
+| 👍/😍 | `:+1:`/`:heart_eyes:` | Praise       | Used when something is well done. No further action required. Should be at least one of these for each review. |
+| ⛏️    | `:pick:`              | Nitpick      | For small details based on preferences. Doesn't block merging. |
+| 🛠️    | `:hammer_and_wrench:` | Suggestion   | To indicate an idea for improvement. By default, the suggestion needs to be accepted or otherwise addressed before merging. |
+| 💅    | `:nail_car:`          | Polish       | To indicate an idea for improvement that isn't fixing something wrong, but just pointing out ways to improve quality. Doesn't block merging by default. |
+| 📋    | `:clipboard:`         | To-do         | For small, necessary changes, such as fixing typos. |
```

Comment:
❓ 
Could we merge that one with the 🧹 ? it feels a bit redundant.

Reply:
Example of a chore (assuming you thought the first draft of #2259 was perfect) would be to rerun the check for the PR so it could pass and merge. (I didn't bother to get the check working, cause I figured I'd have to change it anyway)